README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
```

Comment:
- [ ] OPTIONAL
Also include _package-lock.json_ file in your project directory. This will ensure that your project will run the same on all cloned versions.